app/views/users/profile_edit/_favorite_organizations.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= render "users/profile_sections/profile_section", title: "FAVORITE ORGANIZATIONS"  do %>
+  <div class="user-fav-orgs">
```

Comment:
So for CSS classes, it's recommended to not have bunch of singular vs plural naming to reduce possible "one-off" bugs. Can we make some of those naming more specific? For example if they are list of things you can be like `user-favorite-orgs-list`

Reply:
Can you explain more? I don't see what's wrong with `user-fav-orgs`. I'm also not sure what you mean by "one-off" bugs. Can you give an example?

Reply:
Not to be confused with traditional "off-by-one" errors, I mean to point out the fact that CSS is the number one place to look at wrong selectors. If the selector names are very similar, it's easy to debug or develop on wrong classes.  It's easier to catch on general application languages, but CSS tend to be really painful when one thing is named `user-fav-org` (singular) and another being `user-fav-orgs` (plural).

Reply:
I don't think confusing singular and plural is much of a concern here. This can be said about a lot of other things. But if you feel very strongly about this, I can change it to `user-fav-orgs-list`.

Reply:
I'm only coming from experience, having done many many frontend applications in the past.

CSS is not really a programming language. You can think of it as using tags and depend on them to apply some changes. If tags are way too similarly named, there's a chance of misusing them